app/controllers/api/v1/admin/articles_controller.rb

```diff
@@ -14,8 +14,9 @@ def create
 
   def update
     authorize(current_user)
-
-    Article.update(params[:id], published: params[:article][:published], publisher_id: current_user.id)
+    Article.update(params[:id], published: params[:published], publisher_id: current_user.id)
```

Comment:
Hmm... I'm really not onboard with why you choose to perform this action first and THEN pull the article from the database. Why not do this in one step?

Reply:
Rendering ok response so no need to pull article